pkg/alibabacloud/eni/node_test.go

```diff
@@ -38,41 +40,45 @@ func (e *ENISuite) SetUpTest(c *check.C) {
 
 	metricsapi = metricsmock.NewMockMetrics()
 	alibabaAPI = mock.NewAPI(subnets, vpcs, securityGroups)
-	c.Assert(alibabaAPI, check.Not(check.IsNil))
+	require.NotNil(tb, alibabaAPI)
 	alibabaAPI.UpdateENIs(primaryENIs)
 	instances = NewInstancesManager(alibabaAPI)
-	c.Assert(instances, check.Not(check.IsNil))
-}
+	require.NotNil(tb, instances)
 
-func (e *ENISuite) TearDownTest(c *check.C) {
-	metricsapi = nil
-	alibabaAPI = nil
-	instances = nil
+	tb.Cleanup(func() {
+		metricsapi = nil
+		alibabaAPI = nil
+		instances = nil
+	})
 }
 
-func (e *ENISuite) TestGetMaximumAllocatableIPv4(c *check.C) {
+func TestGetMaximumAllocatableIPv4(t *testing.T) {
+	setup(t)
+
 	n := &Node{}
 	n.k8sObj = newCiliumNode("node", "i-1", "ecs.g7ne.24xlarge", "cn-hangzhou-i", "vpc-1")
-	c.Assert(n.GetMaximumAllocatableIPv4(), check.Equals, 700)
+	require.Equal(t, n.GetMaximumAllocatableIPv4(), 700)
```

Comment:
nit: `require.Equal` wants the 'expected' value first. The test is the same of course, but the error output will say which was the expected and which was the actual value.
```suggestion
	require.Equal(t, 700, n.GetMaximumAllocatableIPv4())
```
I haven't highlighted other uses of `require.Equal` in this file, but I think this is in a few places too. 😅

Reply:
ah thanks, this before I updated my sed commands to swap /1 and /2 :+1:

Reply:
I have updated for other places as well